glean-core/android/src/test/java/mozilla/telemetry/glean/private/StringListMetricTypeTest.kt

```diff
@@ -97,31 +97,6 @@ class StringListMetricTypeTest {
         assertEquals("added1", snapshot2[3])
     }
 
-    @Test
-    fun `lists with no lifetime must not record data`() {
```

Comment:
Just to make sure: this is being removed because it's already covered by Rust tests, right?

Reply:
Sorry to not mention.  It's actually being removed because it's identical to the test below it.